src/azul/service/manifest_service.py

```diff
@@ -705,10 +707,68 @@ def write_to(self, output: IO[str]) -> Optional[str]:
             url = furl(config.service_endpoint(),
                        path=f'/repository/files/{uuid}',
                        args=dict(version=version, catalog=self.catalog))
+            name = self._sanitize_path(name)
             output.write(f'url={self._option(url.url)}\n'
                          f'output={self._option(name)}\n\n')
         return None
 
+    valid_regex_sequence = re.compile(r'''
+        (\.?[^./ ])([^/]*[^./ ])? # match paths not followed or preceeded by '/', ' ', '.' zero or more times
```

Comment:
Try to focus on the intent. Document the why not the how. You don't need to say "match" since that's implied.